frontend/src/app/components/project-list/project-list.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="project-list">
+  <div class="cards-info">
+    <div class="cards-info-title">
+      <h3 class="title">Your organization projects</h3>
+    </div>
+    <div class="cards-info-actions">
+      <tasque-input
+        type="search"
+        placeholder="Search..."
+        [borderRadius]="50"
+        [width]="292"
+      >
```

Comment:
Can you please make search working?
Search on existing project by name

Reply:
Sure, waiting for 'fix input component' pull request and will make search working